content/ember-6-0-released.md

```diff
@@ -0,0 +1,231 @@
+   has more details about how to upgrade your Ember app/addon.
+3. Make sure your app builds successfully.
+4. Resolve all deprecation warnings. These Deprecated APIs are removed in Ember 6.0. You may need to upgrade some of your addon dependencies if they are using deprecated APIs.
+   See the [Ember Deprecation Guide](https://deprecations.emberjs.com/)
+   for more details about specific deprecations and how to resolve them.
+   Applications that need to upgrade through several versions may want to
+   consider isolating individual deprecations by using
+   [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow).
+5. Make sure your app builds successfully and your test suite passes with no
+   deprecations.
+6. Upgrade your app to Ember 6.0. Again,
+   many developers can do this by running
+   `npx ember-cli-update --to 6.0`. Refer to the
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   for more details about how to upgrade your Ember app/addon.
+
+For help or advice along the way, visit [Ember's forums or chat groups](https://emberjs.com/community/).
+
+## The journey towards Ember Polaris
+
+Polaris is the name of an upcoming edition of Ember, first announced at EmberConf 2021. Over the course of 6.x, new features will be added to Ember that will eventually make up the new edition.
+
+Since our major versions tend to be uneventful, Editions are the moment where the Ember community comes together to learn and build in new ways.
+An edition is a set of features, corresponding documentation, and tools that create a cohesive programming model.
+An edition marks a shift and step forward in the framework, beyond what individual features would accomplish.
+Ember Octane was the first new edition that was added to Ember.
+
+Stay tuned for future blog posts with more details and calls for contributors!
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 6.0
+
+Ember.js 6.0 introduces no new public API. Instead, it is comprised of bug fixes and the removal of previously deprecated public API from the 5.x cycle.
+
+#### Bug Fixes
+
+- [#20731](https://github.com/emberjs/ember.js/pull/20731) - Bump barrel file deprecations until 7.0.0
+
+#### Features
+
+Ember.js 6.0 introduced no new features.
+
+#### Deprecations
+
+Ember.js 6.0 introduced 0 deprecations. It removed all deprecations that were
+introduced before 5.10 and slated for removal in 6.0.
+
+Ember 6.0 removes the following features deprecated during 5.x:
+
+- [Removing](https://deprecations.emberjs.com/id/deprecate-implicit-route-model) the implicit route model hook behavior where if no `Route#model` hook was specified and a `_id` parameter was present, Ember would attempt to figure out how to load that model for you.
```

Comment:
This involved an optional feature that can be cleaned up https://github.com/ember-cli/ember-cli/pull/10440 (and probably still needs to be removed from the blueprint though).